user/enterprise/trusty-beta.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Trusty containers for Enterprise (beta)
+layout: en_enterprise
+
+---
+
+**Please note: This is a beta feature. We would definitely like to hear if you run into any problems with the Trusty containers. If you have any questions or run into problems, please write an email to: [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com).**
+
+First of all you need to be running Travis Enterprise 2.1.9 or higher.
+
+1. SSH into the platform machine
+2. Run `travis console`
+3. Then run `Travis::Features.enable_for_all(:template_selection); Travis::Features.enable_for_all(:multi_os)`
+4. You can now exit the console and disconnect from the Travis Enterprise platform machine
+
+**Please be aware that the commands below need to be run on fresh machines. The worker machine(s) with the Precise (Ubuntu 12.04) images can co-exist next to the Trusty workers.**
+
+## AWS
+Spin up AWS worker machines (DeviceMapper installer only supports the recommended c3.2xlarge machines due to the storage layout) and run:
+
+`curl -sSL -o /tmp/installer.sh https://gist.githubusercontent.com/bnferguson/4675ea4fadde8a82613f683fed5f8ad3/raw/trusty-devicemapper-installer.sh`
```

Comment:
Before we release this, we should get aliases like we have for the other worker scripts setup on the enterprise landing page. Going to some random dude's (ha, me) gist isn't confidence inspiring. I don't mind it for a couple direct emails though.

Reply:
Definitely a point on the Todo list. Maybe we can also integrate these scripts into the landing page directly?

Reply:
Yeah I'm into that. I don't like having all of these live on my gist. :)

Reply:
agreed ^^